distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -59,9 +59,10 @@ export EXTRA_JAVA_OPTS
 #
 
 # export JAVA_MIN_MEM # Minimum memory for the JVM
-export JAVA_MAX_MEM=4096M
+# export JAVA_MAX_MEM=4096M
 # export KARAF_HOME # Karaf home folder
 # export KARAF_DATA # Karaf data folder
 # export KARAF_BASE # Karaf base folder
 
 export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+export JAVA_OPTS=-"server -Xms128M -Xmx4096M -XX:+UnlockDiagnosticVMOptions -XX:+UnsyncloadClass -XX:+DisableAttachMechanism"
```

Comment:
Should we assume that the application will _always_ use 4G, or should we just start with 2G and let it grow from there?